### DISCUSSION TITLE
GraphQL: missing field `eventCount` to correctly paginate with `limit: X, offset: Y`

### DISCUSSION THREAD

**Discussion by denisyilmaz** [solspace/craft-calendar#125] [Feature Requests]
When using solspace within GraphQL i cant paginate correctly as i dont get information about available pages.
For built in element types there is a field called  `entryCount`/`categoryCount` etc which returns the total number of entries and so on based on `QueryArguments`

<img width="371" alt="image" src="https://user-images.githubusercontent.com/4189607/164032233-a943277f-e878-43f2-9276-071557303767.png">

It would be great if this field could be implemented. The field should also honor `loadOccurencies` as well so we can differ for different situations where we load all events or only the "base" events.

**Comment by kjmartens** [comment]
Thanks for your feedback, @denisyilmaz. We've added this to our Feature Requests list and will likely address it in an upcoming update. 🙂

**Reply by denisyilmaz** [reply]
Any plans to implement this feature in the near future? We have two bigger projects relying on event pagination in the frontend which is somehow limited when not being able to query for the total count of events (with multiple occurencies).

**Reply by kjmartens** [reply]
Sorry @denisyilmaz, we don't have any plans to address this in the immediate future. 🙁

**Reply by kennethdenhollander** [reply]
@kjmartens Is there an update on this matter already?

**Comment by seandelaney** [comment]
Hopefully resolved in next release

https://github.com/solspace/craft-calendar/pull/422
